pyproject.toml

```diff
@@ -108,8 +108,13 @@ Twitter = "https://twitter.com/qctrlHQ"
 YouTube = "https://www.youtube.com/qctrl"
 GitHub = "https://github.com/qctrl"
 
-# Uncomment the block below if the package requires access to the Internal PyPI.
-# [[tool.poetry.source]]
-# name = "Q-CTRL PyPI"
-# url = "https://pypi.q-ctrl.com/simple"
-# secondary = true
+# Use the internal PyPI as the main source: DO NOT EDIT
+[[tool.poetry.source]]
+name = "Q-CTRL PyPI"
+url = "https://pypi.q-ctrl.com/simple"
+priority = "primary"
+
+# Use the public PyPI as the supplemental source: DO NOT EDIT
+[[tool.poetry.source]]
+name = "PyPI"
+priority = "supplemental"
```

Comment:
Hey @tachikoma-li, perhaps we should mention that for repos that don't require internal packages, public PyPI can still be used as the primary. Thoughts?

Reply:
Good point. Maybe the whole section should remain as a comment as before?

Reply:
Yep I think that works!